cli.js

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env node
 var fs = require('fs')
-var args = require('minimist')(process.argv.slice(2), {boolean: ['prune', 'asar', 'all', 'overwrite', 'strict-ssl']})
+var args = require('minimist')(process.argv.slice(2), {
+  boolean: [
+    'prune',
+    'asar',
+    'all',
+    'overwrite',
+    'strict-ssl',
+    'download.strictSSL'
+  ],
+  default: {
+    'strict-ssl': true,
```

Comment:
This should probably use the `alias` feature of minimist, so the two values are always in sync. Otherwise, the other default might overwrite the user setting.